app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, old_file, new_file = command.split(" ")
```

Comment:
`" "` is the default value for `split`
```suggestion
    command, old_file, new_file = command.split()
```